libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -108,7 +108,7 @@ class MapboxRouteLineView(var options: MapboxRouteLineOptions) {
      */
     fun renderVanishingRouteLineUpdateValue(
```

Comment:
```suggestion
    fun renderRouteLineUpdateValue(
```

Reply:
We should also refactor the docs to say that this is now used to perform various updates to the route line appearance, it's not only the vanishing point anymore.

Reply:
I meant to change the name of that but it slipped though the cracks.